src/github/middleware.ts

```diff
@@ -123,6 +117,12 @@ export default (
 		);
 		for (const subscription of subscriptions) {
 			const { jiraHost } = subscription;
+
+			if (await booleanFlag(BooleanFlags.MAINTENANCE_MODE, false, jiraHost)) {
+				context.log(`Maintenance mode ENABLED for jira host ${jiraHost} - Ignoring event of type ${webhookEvent}`);
+				continue;
+			}
```

Comment:
Why are we putting this in subscriptions if the feature flag is based on the installation?

Reply:
Because the `Installations` table doesn't have the `githubInstallationId`, funnily enough :). So I can't resolve the `jiraHost` via the `Installations` table.